src/components/ButtonWithConfirm/ButtonWithConfirm.js

```diff
@@ -95,30 +113,51 @@ class ButtonWithConfirm extends Component<Props, ComponentState> {
     return (
       <div className='button-with-confirm'>
         {isActive ? (
-          <Fragment>
+          <div
+            className={classNames(
+              'actions',
+              color === 'blue' ? '-white' : '-blue',
+              isReverse && '-reverse',
+              isOverlay && '-overlay',
```

Comment:
we don't need isOverlay prop
just use bg-${color} variant

Reply:
-overley uses additional styles